cybench/models/nn_models.py

```diff
@@ -606,26 +621,105 @@ def fit(
             **fit_params,
         )
 
-    def _separate_ts_static_inputs(self, batch: dict) -> tuple:
-        """Stack time series and static inputs separately.
+    def forward(self, x):
+        for transform in self._transforms:
+            x = transform(x, self._min_date, self._max_date)
+
+        x_ts, x_static = separate_ts_static_inputs(x)
+        x_ts, _ = self._lstm(x_ts)
+        x = torch.cat([x_ts[:, -1, :], x_static], dim=1)
+        output = self._fc(x)
+        return output
+
+
+class ExampleInceptionTime(BaseNNModel):
```

Comment:
I don't like Example in the name. Both LSTM and InceptionTime could be renamed. Baseline is probably better than Example.

Reply:
Let's change it to LSTMModel or LSTM